sda-admin/main.go

```diff
@@ -0,0 +1,455 @@
+  -accession-id ID     Specify the accession ID to assign to the file.
+
+Use 'sda-admin help file <command>' for information on a specific command.`
+
+var fileListUsage = `Usage: sda-admin file list -user USERNAME
+  List all files for a specified user.
+
+Options:
+  -user USERNAME 	Specify the username associated with the files.`
+
+var fileIngestUsage = `Usage: sda-admin file ingest -filepath FILEPATH -user USERNAME
+  Trigger the ingestion of a given file for a specific user.
+
+Options:
+  -filepath FILEPATH   Specify the path of the file to ingest.
+  -user USERNAME       Specify the username associated with the file.`
+
+var fileAccessionUsage = `Usage: sda-admin file set-accession -filepath FILEPATH -user USERNAME -accession-id ACCESSION_ID
+  Assign accession ID to a file and associate it with a user.
+
+Options:
+  -filepath FILEPATH   Specify the path of the file to assign the accession ID.
+  -user USERNAME       Specify the username associated with the file.
+  -accession-id ID     Specify the accession ID to assign to the file.`
+
+var datasetUsage = `Create a dataset:
+  Usage: sda-admin dataset create -dataset-id DATASET_ID [ACCESSION_ID ...]
+    Create a dataset from a list of accession IDs and a dataset ID.
+    
+Release a dataset:
+  Usage: sda-admin dataset release -dataset-id DATASET_ID
+    Release a dataset for downloading based on its dataset ID.
+
+Options:
+  -dataset-id DATASET_ID   Specify the unique identifier for the dataset.
+  [ACCESSION_ID ...]       (For dataset create) Specify one or more accession IDs to include in the dataset.
+
+Use 'sda-admin help dataset <command>' for information on a specific command.`
+
+var datasetCreateUsage = `Usage: sda-admin dataset create -dataset-id DATASET_ID [ACCESSION_ID ...]
+  Create a dataset from a list of accession IDs and a dataset ID.
+
+Options:
+  -dataset-id DATASET_ID    Specify the unique identifier for the dataset.
+  [ACCESSION_ID ...]         (For dataset create) Specify one or more accession IDs to include in the dataset.`
+
+var datasetReleaseUsage = `Usage: sda-admin dataset release -dataset-id DATASET_ID
+  Release a dataset for downloading based on its dataset ID.
+
+Options:
+  -dataset-id DATASET_ID    Specify the unique identifier for the dataset.`
+
+var versionUsage = `Usage: sda-admin version
+  Show the version information for sda-admin.`
+
+func printVersion() {
+	fmt.Printf("sda-admin version %s\n", version)
+}
+
+func parseFlagsAndEnv() error {
+	// Set up flags
+	flag.StringVar(&apiURI, "uri", "", "Set the URI for the SDA server (optional if API_HOST is set)")
+	flag.StringVar(&token, "token", "", "Set the authentication token (optional if ACCESS_TOKEN is set)")
+
+	// Custom usage message
+	flag.Usage = func() {
+		fmt.Println(usage)
+	}
+
+	// Parse global flags first
+	flag.Parse()
+
+	// If no command is provided, show usage
+	if flag.NArg() == 0 {
+		return errors.New(usage)
+	}
+
+	if flag.Arg(0) == "help" {
```

Comment:
```suggestion
	if flag.Arg(0) == "help" || flag.Arg(0) == "version" {
```

Version is also a special case.